File: src/main/java/com/michael/Hello.java
package com.michael;

public class Hello {
    public static void main(String[] args) {
        //System.out.println("Hello World");
        Person person=new Person();
        person.hello();
        int age=19; //short 16bits;int 32bits;long 64bits;
        Integer age2=19;//參造資料型態;包裝類別
        char c='A';//16bits
        Character c2='A';
        byte b=120;//8bits
        float weight=66.5f;//32bits後面要加f
        double height=1.7;//64bits
        boolean adult=true;
        boolean enroll=false;
        String name="Michael";
    }
}

File: src/main/kotlin/com/michael/Hello.kt
package com.michael

fun main(args: Array<String>) {
    //println("Hello KT")
    //Human().hello()
    val h=Human()
    h.hello()
    val age=19 //變數型態的自動推斷 val不可變 var可變
    var weight=66.5
    var name:String
    name = "Michael"
}

class Human{
    fun hello(){
        println("Hello KT")
    }
}